README.md

```diff
@@ -50,3 +50,15 @@ Delete @elyra/canvas from node_modules of your application
 # Run npm install to get local copy of common-canvas and/or common-properties
 npm install
 ```
+
+### Contribute to the Elyra Canvas documentation (mkdocs)
+1. Python v3 needs to be available.
```

Comment:
Add a step - 
2. Go to `canvas` directory
3. Install required mkdocs packages using pip3...